config_templates/gretel/tasks/transform__default.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: "1.0"
+name: default
+task:
+  name: transform
+  config:
+      globals:
+        classify:
+          enable: true
```

Comment:
note to self: disable for now until we land ner support

Reply:
this is supported now, OK to land as-is